scrapi/base/__init__.py

```diff
@@ -65,14 +93,20 @@ class OAIHarvester(BaseHarvester):
 
     record_encoding = None
 
-    def __init__(self, name, base_url, timezone_granularity=False, timeout=0.5, property_list=None, approved_sets=None):
-        self.name = name
-        self.base_url = base_url
-        self.property_list = property_list or ['date', 'language', 'type']
-        self.approved_sets = approved_sets
+    def __init__(self, timezone_granularity=False, timeout=0.5, property_list=None, approved_sets=None):
+        # self.name = name
+        # self.base_url = base_url
         self.timeout = timeout
         self.timezone_granularity = timezone_granularity
 
+    @property
+    def property_list(self):
+        return ['date', 'language', 'type']
```

Comment:
This is probably just a WIP thing, but just noting to make sure this doesn't stay